### DISCUSSION TITLE
Pointers to hosting 3kh0.net copy or homemade proxy on home laptop

### DISCUSSION THREAD

**Discussion by insertcash** [3kh0/ext-remover#730] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
I'm trying to host my own proxy for personal reasons, but I don't know where to start. I'm trying to find a place to host it for free because if it gets blocked I can't be tied down to a singular link. Any tips on hosting a proxy and how to make one?

**Comment by Mazedotexe** [comment]
no idea what you should do for the proxy, but as for website hosting options like what you want, u should use netlify or glitch

**Comment by Su4o** [comment, marked as answer]
(im guessing your talking about a school unblocked proxy, like ludicrous.icu) 
 
Well here are some sites you can used (almost all are free)

Replit
CodeSandbox 
Stackblitz 
Vercel
Netlify
Glitch
Koyeb 
Vs Code
Codeanywhere
Terminal (localhost) (if you know how to)
-------------------
